core/server/api/v2/utils/validators/input/schemas/tags.json

```diff
@@ -0,0 +1,63 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "tags",
+    "title": "tags",
+    "description": "Base tags definitions",
+    "definitions": {
+        "tag": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "maxLength": 191,
+                    "pattern": "^([^,]|$)"
+                },
+                "slug": {
+                    "type": ["string", "null"],
+                    "maxLength": 191
+                },
+                "description": {
+                    "type": ["string", "null"],
+                    "maxLength": 500
+                },
+                "feature_image": {
+                    "type": ["string", "null"],
+                    "format": "uri-reference",
+                    "maxLength": 2000
+                },
+                "visibility": {
+                    "type": "string",
+                    "enum": ["public", "internal"]
+                },
+                "meta_title": {
+                    "type": ["string", "null"],
+                    "maxLength": 300
+                },
+                "meta_description": {
+                    "type": ["string", "null"],
+                    "maxLength": 500
+                },
+                "id": {
+                    "strip": true
+                },
+                "parent": {
```

Comment:
we probably also want to ignore "parent_id" (db property), because "parent" was invented in early Ghost ages to map "parent_id" to "parent", but it was never used 🤩